src/Symfony/Component/Mailer/Bridge/Mailgun/RemoteEvent/MailgunPayloadConverter.php

```diff
@@ -19,6 +19,13 @@
 
 final class MailgunPayloadConverter implements PayloadConverterInterface
 {
+    public const MAILGUN_SPECIFIC_DROPPED_CODES = [
```

Comment:
this should be a private constant, not a public one

Reply:
You are right, habit from our code where we reference it in the test class!